TombEngine/Game/Lara/lara_struct.h

```diff
@@ -1263,12 +1263,12 @@ struct LaraControlData
 	WeaponControlData	 Weapon	   = {};
 
 	bool IsClimbingLadder = false;
-	bool Locked			  = false; // IsLocked
-	bool IsLow			  = false;
+	bool IsLocked		  = false;
 	bool IsMonkeySwinging = false;
 	bool IsMoving		  = false;
-	bool RunJumpQueued	  = false; // IsRunJumpQueued
-	bool KeepLow		  = false; // IsInLowSpace
+	bool IsRunJumpQueued  = false;
+	bool IsInLowPosition  = false;
```

Comment:
I don't agree with renaming `KeepLow` to `IsInLowSpace`. Before, it was clear that `IsLow` means *current* Lara state, while `KeepLow` means that she *should remain in low state*. Now it got obscure and I can't really understand how `IsInLowPosition` differs from `IsInLowSpace`.